src/Api/SecretsManager/Models/Request/AccessTokenCreateRequestModel.cs

```diff
@@ -34,4 +34,13 @@ public ApiKey ToApiKey(Guid serviceAccountId)
             EncryptedPayload = EncryptedPayload,
         };
     }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+    {
+        if (ExpireAt <= DateTime.Now)
```

Comment:
```suggestion
        if (ExpireAt <= DateTime.UtcNow)
```

We send and handle time as UTC.

I would also add some logic here to handle the null case for when we are doing never expiring tokens.